diff --git a/tmux-sessionizer b/tmux-sessionizer
index fa1bec5..4d79e0b 100755
--- a/tmux-sessionizer
+++ b/tmux-sessionizer
@@ -22,9 +22,7 @@ hydrate() {
 if [[ $# -eq 1 ]]; then
     selected=$1
 else
-    # If someone wants to make this extensible, i'll accept
-    # PR
-    selected=$(find ~/ ~/personal ~/personal/dev/env/.config -mindepth 1 -maxdepth 1 -type d | fzf)
+    selected=$(find -L "${XDG_REPOS_DIR}" "${XDG_REPOS_DIR}"/* "${XDG_DOCUMENTS_DIR}" -maxdepth 1 -mindepth 1 -type d ! -name ".*" ! -name ".old" ! -name "_archive" | sort | fzf)
 fi
 
 if [[ -z $selected ]]; then
