diff --git a/tmux-sessionizer b/tmux-sessionizer
index 3e607d7..27729b1 100755
--- a/tmux-sessionizer
+++ b/tmux-sessionizer
@@ -22,7 +22,7 @@ hydrate() {
 if [[ $# -eq 1 ]]; then
     selected=$1
 else
-    selected=$(find $(eval echo $(xargs < "$XDG_CONFIG_HOME/tmux-sessionizer/directories")) -mindepth 1 -maxdepth 1 -type d | sed "s|$HOME/||" | fzf)
+    selected=$(find $(eval echo $(xargs < "$XDG_CONFIG_HOME/tmux-sessionizer/directories")) -mindepth 1 -maxdepth 1 -type d | sed "s|$HOME/||" | sort | fzf)
 fi
 
 if [[ -z $selected ]]; then
