diff --git a/tmux-sessionizer b/tmux-sessionizer
index fa1bec5..c9094e7 100755
--- a/tmux-sessionizer
+++ b/tmux-sessionizer
@@ -12,7 +12,9 @@ has_session() {
 }
 
 hydrate() {
-    if [ -f $2/.tmux-sessionizer ]; then
+    if [ -f $2/.tmux ]; then
+        tmux send-keys -t $1 "source $2/.tmux" c-M
+    elif [ -f $2/.tmux-sessionizer ]; then
         tmux send-keys -t $1 "source $2/.tmux-sessionizer" c-M
     elif [ -f $HOME/.tmux-sessionizer ]; then
         tmux send-keys -t $1 "source $HOME/.tmux-sessionizer" c-M
