diff --git a/tmux-sessionizer b/tmux-sessionizer
index fa1bec5..536d4b0 100755
--- a/tmux-sessionizer
+++ b/tmux-sessionizer
@@ -14,8 +14,8 @@ has_session() {
 hydrate() {
     if [ -f $2/.tmux-sessionizer ]; then
         tmux send-keys -t $1 "source $2/.tmux-sessionizer" c-M
-    elif [ -f $HOME/.tmux-sessionizer ]; then
-        tmux send-keys -t $1 "source $HOME/.tmux-sessionizer" c-M
+    elif [ -f "$XDG_CONFIG_HOME/tmux-sessionizer/default" ]; then
+        tmux send-keys -t $1 "source $XDG_CONFIG_HOME/tmux-sessionizer/default" c-M
     fi
 }
 
