diff --git a/config.def.h b/config.def.h
index 9efa774..eb69e91 100644
--- a/config.def.h
+++ b/config.def.h
@@ -60,9 +60,14 @@ static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn()
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
 static const char *termcmd[]  = { "st", NULL };
 
+/* custom commands */
+static const char *firefox[]  = { "firefox", NULL };
+static const char *rofi[]  = { "rofi", "-show", "drun", NULL };
+static const char *thunar[]  = { "thunar", NULL };
+
 static const Key keys[] = {
 	/* modifier                     key        function        argument */
-	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_p,      spawn,          {.v = rofi } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
@@ -95,6 +100,8 @@ static const Key keys[] = {
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY|ShiftMask,             XK_b,      spawn,          {.v = firefox } },
+	{ MODKEY|ShiftMask,             XK_f,      spawn,          {.v = thunar } },
 };
 
 /* button definitions */
