diff --git a/config.def.h b/config.def.h
index 9efa774..dd6779a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -95,6 +95,11 @@ static const Key keys[] = {
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY|ShiftMask,             XK_b,      spawn,          SHCMD("firefox") },
+	{ MODKEY|ShiftMask,             XK_f,      spawn,          SHCMD("thunar") },
+	{ MODKEY|ShiftMask,             XK_s,      spawn,          SHCMD("flameshot gui") },
+	{ MODKEY|ShiftMask,             XK_y,      spawn,          SHCMD("copyq toggle") },
+	{ MODKEY,                       XK_Insert, spawn,          SHCMD("xdotool type $(grep -v '^#' ~/snippets.txt | sort | dmenu -i -l 50 | cut -d' ' -f1)") },
 };
 
 /* button definitions */
