diff --git a/.local/bin/displayselect b/.local/bin/displayselect
index 51dd468..f69f154 100755
--- a/.local/bin/displayselect
+++ b/.local/bin/displayselect
@@ -57,7 +57,6 @@ onescreen() { # If only one output available or chosen.
 	}
 
 postrun() { # Stuff to run to clean up.
-	setbg		# Fix background if screen size/arangement has changed.
 	{ killall dunst ; setsid -f dunst ;} >/dev/null 2>&1 # Restart dunst to ensure proper location on screen
 	}
 
